lib/post_install.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable */
+// adapted based on rackt/history (MIT)
+// Node 4+
+var execSync = require('child_process').execSync;
+var stat = require('fs').stat;
+
+function exec(command) {
+  execSync(command, {
+    stdio: [0, 1, 2]
+  });
+}
+
+stat('dist', function(error, stat) {
+  // Skip building on Travis
+  if (process.env.TRAVIS) {
+    return;
+  }
+
+  if (error || !stat.isDirectory()) {
+    // Only build deps
+    exec('npm i babel-cli babel-jest babel-preset-webpack cross-env del-cli');
```

Comment:
What if we do `npm install --only=dev` here instead of copypasting dependencies?